styles.css

```diff
@@ -552,12 +675,53 @@ p {
     padding: 5rem 9rem;
   }
 
+  .skills img {
+    width: 48px;
+    height: 48px;
+  }
+
+  .link-button a img {
+    margin-left: 5px;
+  }
+
+  .popup-page .snap-image,
+  .popup-page .snap-image img {
+    height: 360px;
+    backdrop-filter: blur(5px);
+  }
+
+  .popup-page .card {
+    width: 80%;
+    display: flex;
+    margin: 0 auto;
+    margin-top: 10%;
```

Comment:
I see that you have tried to fix the issue with the popup modal, but it still doesn't show all the contents. I think the problem is that you have set the margin of the popup container to 10%, which means it will take up the whole viewport height. However, since your popup has a margin-top of 10%, it will overflow the viewport by that amount and hide some of the contents.

One possible solution is to reduce the height of the popup container by 10%, so it fits within the viewport. You can do this by using the calc() function in CSS, like this:

```css
.popup-container {
  height: calc(100% - 10%);
  overflow: scroll;
}
```

This way, you can see all the contents of the popup and also match the Figma design. 👍

I hope this helps you improve your project. Keep up the good work! 😊